jms/org.wso2.carbon.transport.jms/src/main/resources/META-INF/services/org.wso2.carbon.messaging.ServerConnector

```diff
@@ -0,0 +1 @@
+org.wso2.carbon.transport.jms.listener.JMSServerConnector
```

Comment:
this file is not needed as we only load server connector providers via SPI

Reply:
This is fixed.